owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/model/RangeModel.java

```diff
@@ -0,0 +1,65 @@
+package fr.openwide.core.wicket.more.model;
+
+import org.apache.wicket.model.AbstractReadOnlyModel;
+import org.apache.wicket.model.IModel;
+
+import com.google.common.collect.Range;
+
+import fr.openwide.core.wicket.more.util.model.Detachables;
+
+/**
+ * Creates a range model representing an interval between two values.
+ */
+public class RangeModel<C extends Comparable<?>> extends AbstractReadOnlyModel<Range<C>> {
+
+	private static final long serialVersionUID = 2798667461907515108L;
+	
+	private final IModel<C> lowerBoundModel;
+	private final IModel<C> upperBoundModel;
+	
+	/**
+	 * Constructs a closed range model from a lower and an upper bound.
+	 * 
+	 * @param lowerBoundModel
+	 *         the lower bound of this range, or null for negative infinity
+	 * @param upperBoundModel
+	 *         the upper bound of this range, or null for positive infinity
+	 */
+	public RangeModel(IModel<C> lowerBoundModel, IModel<C> upperBoundModel) {
```

Comment:
It's just a minor detail, but could you use `IModel<? extends C>` instead of `IModel<C>` here, please? Same goes for the class attributes (`lowerBoundModel`/`upperBoundModel`).
Since the models are only used to read from them, there's no strong requirement that their type parameter be *exactly* `C`, it may be a subclass as well.